tools/csv-generator/csv-generator.go

```diff
@@ -43,12 +45,9 @@ func main() {
 	}
 
 	if *dumpCRDs {
-		vmiCrds := vmioperator.NewCrds()
```

Comment:
Please remove `NewCrds` from the `operator` package; it does not seem to be used anywhere anymore.

Reply:
It is part of not complete refactoring. Will remove.